src/WebApp/Program.cs

```diff
@@ -1,12 +1,26 @@
 ﻿using eShop.WebApp.Components;
 using eShop.ServiceDefaults;
 
+// Enable SK Telemetry
+AppContext.SetSwitch("Microsoft.SemanticKernel.Experimental.GenAI.EnableOTelDiagnosticsSensitive", true);
+
 var builder = WebApplication.CreateBuilder(args);
 
 builder.AddServiceDefaults();
 
 builder.Services.AddRazorComponents().AddInteractiveServerComponents();
 
+// Configure SK Telemetry
+builder.Services.AddOpenTelemetry()
+    .WithMetrics(m =>
+    {
+        m.AddMeter("Microsoft.SemanticKernel*");
+    })
+    .WithTracing(t =>
+    {
+        t.AddSource("Microsoft.SemanticKernel*");
+    });
```

Comment:
Should all of these changes go into service defaults where the rest of telemetry is configured?

Reply:
Good question. We can do it there as well, but AFAIK we're mainly surfacing the information from the web app. Not sure if we'd get additional info if it was baked into service defaults.